data/doveadm.js

```diff
@@ -2075,195 +2376,257 @@ returned.`,
 			},
 			'all-namespaces': {
 				cli: 'N',
 				type: doveadm_arg_types.BOOL,
+				text: `Sync all namespaces.`,
 			},
 			'exclude-mailbox': {
 				cli: 'x',
+				example: ['Spam'],
 				type: doveadm_arg_types.ARRAY,
+				text: `Exclude these mailboxes.`,
 			},
 			'all-mailbox': {
 				cli: 'a',
+				example: 'AllMailbox',
 				type: doveadm_arg_types.STRING,
+				text: `Name for the "All Mails" virtual mailbox.`,
 			},
 			state: {
 				cli: 's',
+				example: 'state_string',
 				type: doveadm_arg_types.STRING,
+				text: `Use stateful sync.`,
 			},
 			'sync-since-time': {
 				cli: 't',
+				example: '2023-04-13',
 				type: doveadm_arg_types.STRING,
+				text: `Skip mails older than this date.`,
 			},
 			'sync-until-time': {
 				cli: 'e',
+				example: '2023-10-21',
 				type: doveadm_arg_types.STRING,
+				text: `Skip mails newer than this date.`,
 			},
 			'sync-flags': {
 				cli: 'O',
+				example: '\\Flagged',
 				type: doveadm_arg_types.STRING,
+				text: `Sync only mails with this flag.`,
 			},
 			'sync-max-size': {
 				cli: 'I',
+				example: '10M',
 				type: doveadm_arg_types.STRING,
+				text: `Skip mails larger than this size.`,
 			},
 			timeout: {
 				cli: 'T',
+				example: 600,
 				type: doveadm_arg_types.INTEGER,
+				text: `Timeout for stalled I/O operations.`,
 			},
 			'default-destination': {
 				cli: 'd',
 				type: doveadm_arg_types.BOOL,
+				text: `Use the default destination.`,
 			},
+			/* This is not documented in man page, so it is hidden here.
 			'legacy-dsync': {
 				cli: 'E',
 				type: doveadm_arg_types.BOOL,
+				text: `Do full synchronization.`,
 			},
+			*/
 			'oneway-sync': {
 				cli: '1',
 				type: doveadm_arg_types.BOOL,
+				text: `Do one-way synchronization.`,
 			},
 			destination: {
+				example: ['maildir:~/Maildir'],
 				positional: true,
 				type: doveadm_arg_types.ARRAY,
+				text: `Sync destinations.`
 			},
 		},
 		flags: doveadm_flag_types.USER | doveadm_flag_types.USERFILE,
-		man: 'doveadm-sync'
+		man: 'doveadm-sync',
+		text: `Dovecot's mailbox synchronization utility.`,
 	},
 
 	user: {
 		args: {
 			'socket-path': {
 				cli: 'a',
 				cli_only: true,
+				example: `/var/run/dovecot/doveadm-server`,
 				type: doveadm_arg_types.STRING,
 				text: `userdb socket path`
 			},
 			'auth-info': {
 				cli: 'x',
-				type: doveadm_arg_types.ARRAY
+				example: ['service=imap'],
+				type: doveadm_arg_types.ARRAY,
+				text: `Additional conditions for auth lookup.`,
 			},
 			field: {
 				cli: 'f',
-				type: doveadm_arg_types.STRING
+				example: 'userdb_field',
+				type: doveadm_arg_types.STRING,
+				text: `Only show value of this field.`,
 			},
+			// TODO: Needs documentation in man file
 			'expand-field': {
 				cli: 'e',
 				type: doveadm_arg_types.STRING
 			},
 			'userdb-only': {
 				cli: 'u',
-				type: doveadm_arg_types.BOOL
+				type: doveadm_arg_types.BOOL,
+				text: `Only show values from userdb.`,
```

Comment:
do we want to consistently use ? with boolean?

Reply:
Yes, I've removed all the question marks in here.